### ISSUE TITLE
Make GHG Center top visual priority on home page

### ISSUE THREAD

**Issue by j08lue** [NASA-IMPACT/veda-config-eic#75]
For the official GHG Center announcement, Earth.gov should feature the GHG Center much more prominently than it currently is (a good deal down the home page). A suggestion was to move the current banner up the page.

I would ask @faustoperez for guidance on how to do this, but please note that this needs to be done immediately (before presentation early on Monday).

Pinging @agurvich for visibility. This was a request from program leadership - we can discuss any questions to this directive separately.

## Acceptance criteria

- [x] GHG Center visually more prominent on Earth.gov landing page

**Comment by agurvich** [comment]
Also, the specific ask was for the GHG banner to be moved to the top of the page rather than the bottom. I was thinking before the themes but after the interagency partners. It doesn't have to be permanent but I think it would satisfy the request and we can think about how best to highlight GHG Center in the long run.

**Comment by faustoperez** [comment]
> Also, the specific ask was for the GHG banner to be moved to the top of the page rather than the bottom. I was thinking before the themes but after the interagency partners

Here's an updated homepage mockup.
Link to Figma file 👉 https://www.figma.com/file/2Vd0SDhFOTBJaYioJPr0Ei/Earth.gov?type=design&node-id=730%3A3033&mode=design&t=CbgzBDnFSIZyt9aE-1

<img width="1440" alt="Homepage v3" src="https://github.com/NASA-IMPACT/veda-config-eic/assets/12268629/851c021d-4c0d-4535-9d6b-a65df8fcfc22">

**Comment by agurvich** [comment]
Agreed, this looks good to me as well. 

When I tried this on my local build I moved the gradient block up with it and it looked weird but ditching the gradient altogether helps make it less jarring. 

Once we have multiple centers (however long down the road) we should revisit how we want to make them easy to access from the home page (at the moment I'm partial to the featured theme card slider approach but we should have plenty of time to think of an alternative).
